Assets: Only supports 100 named groups

You can only add about 100 assets explicitly. If you add one more the error message "AssertionError: sorry, but this version only supports 100 named groups" appears.

It seems to be an behaviour of python regular expressions.

Test file Main.js of html5 boilerplate skelleton:

``` javascript
/**
// Adding the 100th assets failed
#asset(100.png)

#asset(1.png)
#asset(2.png)
#asset(3.png)
#asset(4.png)
#asset(5.png)
#asset(6.png)
#asset(7.png)
#asset(8.png)
#asset(9.png)
#asset(10.png)
#asset(11.png)
#asset(12.png)
#asset(13.png)
#asset(14.png)
#asset(15.png)
#asset(16.png)
#asset(17.png)
#asset(18.png)
#asset(19.png)
#asset(20.png)
#asset(21.png)
#asset(22.png)
#asset(23.png)
#asset(24.png)
#asset(25.png)
#asset(26.png)
#asset(27.png)
#asset(28.png)
#asset(29.png)
#asset(30.png)
#asset(31.png)
#asset(32.png)
#asset(33.png)
#asset(34.png)
#asset(35.png)
#asset(36.png)
#asset(37.png)
#asset(38.png)
#asset(39.png)
#asset(40.png)
#asset(41.png)
#asset(42.png)
#asset(43.png)
#asset(44.png)
#asset(45.png)
#asset(46.png)
#asset(47.png)
#asset(48.png)
#asset(49.png)
#asset(50.png)
#asset(51.png)
#asset(52.png)
#asset(53.png)
#asset(54.png)
#asset(55.png)
#asset(56.png)
#asset(57.png)
#asset(58.png)
#asset(59.png)
#asset(60.png)
#asset(61.png)
#asset(62.png)
#asset(63.png)
#asset(64.png)
#asset(65.png)
#asset(66.png)
#asset(67.png)
#asset(68.png)
#asset(69.png)
#asset(70.png)
#asset(71.png)
#asset(72.png)
#asset(73.png)
#asset(74.png)
#asset(75.png)
#asset(76.png)
#asset(77.png)
#asset(78.png)
#asset(79.png)
#asset(80.png)
#asset(81.png)
#asset(82.png)
#asset(83.png)
#asset(84.png)
#asset(85.png)
#asset(86.png)
#asset(87.png)
#asset(88.png)
#asset(89.png)
#asset(90.png)
#asset(91.png)
#asset(92.png)
#asset(93.png)
#asset(94.png)
#asset(95.png)
#asset(96.png)
#asset(97.png)
#asset(98.png)
#asset(99.png)
*/

core.Class("myboilerplateproject.Main", {

    construct: function() {

        alert("Hello from myboilerplateproject!");

    },

    members: {


    }

});
```

--- a/jasy/asset/Manager.py
+++ b/jasy/asset/Manager.py
@@ -350,7 +350,7 @@ class AssetManager:
             hints.update(classObj.getMetaData(self.__session.getCurrentPermutation()).assets)
         
         # Compile filter expressions
-        matcher = "^%s$" % "|".join(["(%s)" % fnmatch.translate(hint) for hint in hints])
+        matcher = "^%s$" % "|".join(["(?:%s)" % fnmatch.translate(hint) for hint in hints])
         Console.debug("Compiled asset matcher: %s" % matcher)
         
         return re.compile(matcher)

